tests/data_context/test_get_data_context.py

```diff
@@ -158,6 +159,10 @@ def test_cloud_context_disabled(set_up_cloud_envs, tmp_path: pathlib.Path):
 def test_cloud_missing_env_throws_exception(
     monkeypatch, empty_ge_cloud_data_context_config
 ):
+    # Delete local env vars (if present)
+    for env_var in GXCloudEnvironmentVariable:
+        monkeypatch.delenv(env_var, raising=False)
+
```

Comment:
Test fails if you have env vars set - let's ensure we clear out our environment for consistent behavior

Reply:
This seems like something we might want to re-use in a fixture.
`clean_cloud_env_vars` / `no_cloud_env` 🤷

Reply:
yup this was done shortly after!